packages/icons-react16/package.json

```
@@ -0,0 +1,63 @@
+{
+    "name": "@hopper-ui/icons-react16",
+    "author": "Workleap",
+    "version": "1.0.2",
+    "description": "The icons package, that was created to target React 16.",
```

You shouldn't need a comma here, you could even shorten it to this:
```suggestion
    "description": "The icons package that targets React 16.",
```